public/js/index.js

```diff
@@ -0,0 +1,17 @@
+const addNote = document.getElementById('addNote');
+addNote.addEventListener('click', () => {
+    console.log("Updating...");
+    const tit = document.getElementById('title').value;
+    const desc = document.getElementById('description').value;
+
+    if(localStorage.getItem('itemsJson') == null){
```

Comment:
Keep consistency in code format. Add spaces around if and else.
if-else code format should be like below
`
if (...) {
    ...
} else {
    ... 
}
`